### DISCUSSION TITLE
Apresente-se aqui.

### DISCUSSION THREAD

**Discussion by elenderg** [elenderg/Portugues-Puro#1] [Comunicados Importantes]
## 👋 Seja Bem-vindo!
  Estamos usando o Github Discussions como um lugar para nos conectar com outros membros de nossa comunidade. Esperamos que você:
  * Faça perguntas sobre as quais está curioso.
  * Compartilhe idéias.
  * Interaja com outros membros da comunidade.
  * Dê as boas-vindas aos outros e seja mente aberta. Lembre-se de que esta é uma comunidade que construímos juntos 💪.

  Para começar, comente abaixo com uma apresentação sobre você e nos conte o que você faz nesta comunidade..

**Comment by fnandoleal** [comment]
A extensão no vscode é pra mudar os comandos do inglês pra português no vscode?

**Reply by elenderg** [reply]
Não. Para alterar os comandos do VS code para português, basta instalar a extensão oficial da Microsoft, disponível [aqui](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-pt-BR):
[https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-pt-BR](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-pt-BR)

**Reply by fnandoleal** [reply]
Não me refiro a interface do vs code. Sei que tem como traduzir. Refiro aos comandos que normalmente se usa inglês na linguagem escolhida. Tem alguns projetos de linguagem de programação que se traduzem tais termos do inglês. Linguagem Égua, Portugol, Visualg são exemplos.

**Comment by elenderg** [comment]
Sim, mas apenas em certo sentido.  
Portugol é um termo dado para um conjunto de linguagens de programação diferentes. Por exemplo:

- A linguagem usada no [VisuAlg](https://dicasdeprogramacao.com.br/download-visualg/), cuja sintaxe se assemelha em certos pontos à [Pascal](https://www.freepascal.org/)
- A linguagem usada no [Portugol Studio](https://univali-lite.github.io/Portugol-Studio/), cuja sintaxe se assemelha em certos pontos à [Java](https://www.java.com/pt-BR/) e [C#](https://dotnet.microsoft.com/pt-br/languages/csharp)

A [Linguagem Égua](https://egua.dev/) (e seu dialeto, [Delégua](https://github.com/DesignLiquido/delegua?tab=readme-ov-file)) são linguagens de programação que buscam "traduzir" os comandos da linguagem [Javascript](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript) para Português.
Na verdade a empresa [Design Líquido](https://github.com/DesignLiquido), responsável pela Delégua também criou projetos semelhantes para:

- [HTML](https://developer.mozilla.org/pt-BR/docs/Web/HTML) (confira o repositório [LMHT](https://github.com/DesignLiquido/LMHT)), 
- [CSS](https://developer.mozilla.org/pt-BR/docs/Web/CSS) (confira o repositório [FolEs](https://github.com/DesignLiquido/FolEs)); e
- [SQL](https://pt.wikipedia.org/wiki/SQL) (confira o repositório [LinConEs](https://github.com/DesignLiquido/LinConEs)).

Neste sentido, poderíamos dizer que [Português Puro](https://github.com/elenderg/Portugues-Puro) nada mais é do que uma versão da Linguagem de Programação denominada "[Plain English](https://rosettacode.org/wiki/Category:Plain_English)".

No entanto, a equipe de desenvolvimento do Português Puro está buscando adicionar funcionalidades extras à linguagem (como suporte nativo a arrays, por exemplo), bem como a todo o ambiente de desenvolvimento. 
Já desenvolvemos um compilador que funciona via linha de comando e estamos em vias de implementar um Servidor de Linguagem para o Português Puro.
Também estamos planejando o desenvolvimento de um depurador (debugger) e, no futuro faremos o port de tudo isso para o Linux. 
Assim como [Gerry Rzeppa](https://www.quora.com/profile/Gerry-Rzeppa), um dos criadores do Plain English, disse:

> Português Puro é um projeto totalmente original, no sentido em que já se encontra totalmente distinguível da linguagem original, com seus próprios termos e convenções de estilo.

**Comment by nataliapastre-dev** [comment]
Olá, pessoal! 👋

Meu nome é Natália e estou em transição de carreira para a área de tecnologia. Atualmente estudo Análise e Desenvolvimento de Sistemas e venho me especializando em desenvolvimento Front-end, principalmente com React, JavaScript e TypeScript.

Também tenho estudado Back-end, Python, Java, SQL e APIs, buscando ampliar cada vez mais meus conhecimentos.

Estou aqui para aprender, trocar experiências, acompanhar projetos e, sempre que possível, contribuir com a comunidade. 🚀

💻 GitHub: https://github.com/nataliapastre-dev
🔗 LinkedIn: https://www.linkedin.com/in/nataliapastre-dev/

Espero poder aprender bastante com vocês e também compartilhar um pouco da minha jornada! 😊
